arm-graphrbac/1.6/swagger/graphrbac.json

```diff
@@ -226,7 +226,153 @@
           }
         }
       }
+    },	
+    "/{tenantID}/applications/{applicationObjectId}/keyCredentials": {
+      "get": {
+        "tags": [
+          "ApplicationKeyCredentials"
+        ],
+        "operationId": "Applications_ListKeyCredentials",
+        "description": "Get keyCredentials associated with the application by object Id. Reference: https://msdn.microsoft.com/en-us/library/azure/ad/graph/api/entity-and-complex-type-reference#keycredential-type",
+        "parameters": [
+          {
+            "name": "applicationObjectId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Application object id",
+            "x-ms-skip-url-encoding": true
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/tenantIDInPath"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "",
+            "schema": {
+              "$ref": "#/definitions/KeyCredentialListResult"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      },
+      "patch": {
+        "tags": [
+          "ApplicationKeyCredentials"
+        ],
+        "operationId": "Applications_PatchKeyCredentials",
```

Comment:
Applications_**Update**KeyCredentials. This would be more consistent with other APIs, rather than using the rest verb patch.